Sources/Hypertext.swift

```diff
@@ -6,18 +6,23 @@
 //  Copyright © 2016 Sahand Nayebaziz. All rights reserved.
 //
 
-public protocol Renderable {
-    func render() -> String
-    func render(startingWithSpacesCount: Int) -> String
+public protocol Renderable: CustomStringConvertible {
+    func render(indent: Int?, startingWithSpacesCount: Int) -> String
```

Comment:
@sahandnayebaziz the `indent` argument has the purpose to give the developer the choice how big they want their indentation to be. I usually write HTML with 4 spaces indentation and I saw that you chose 2 spaces, so I thought why not give the developer the choice.

I chose to make `indent` optional, because then you can convey you want a minified output by supplying `nil` for `indent`. Or not supplying it at all of course, because then `nil` is assumed.

Does that answer your question?